demos/collection/card-deck.php

```diff
@@ -14,13 +14,13 @@
 \Atk4\Ui\Header::addTo($app, ['Card Deck', 'size' => 1, 'subHeader' => 'Card can be display in a deck, also using model action.']);
 
 $countries = new Country($app->db);
-$countries->addCalculatedField('Cost', function ($model) {
+$countries->addCalculatedField('Cost', function (Country $country) {
     return '$ ' . number_format(random_int(500, 1500));
 });
 
 $action = $countries->addUserAction('book', [
-    'callback' => function ($model, $email, $city) {
-        return 'Your request to visit ' . ucwords($city) . ' in ' . $model->get('name') . ' was sent to: ' . $email;
+    'callback' => function (Country $country, $email, $city) {
+        return 'Your request to visit ' . ucwords($city) . ' in ' . $country->name . ' was sent to: ' . $email;
```

Comment:
Shouldn't that be `$country->fieldName()->name`?

Reply:
no ;-)